papaparse.js

```diff
@@ -326,9 +326,10 @@
 				return;
 
 			if (typeof _config.delimiter === 'string'
-				&& _config.delimiter.length === 1
-				&& Papa.BAD_DELIMITERS.indexOf(_config.delimiter) === -1)
+                && !Papa.BAD_DELIMITERS.filter(function(value) { return _config.delimiter.indexOf(value) !== -1; }).length)
 			{
+				// We have a string as config and it does not contains any
```

Comment:
Is really needed to add this comment?

Reply:
nope. is not needed...but that whole logic is pretty long and I wanted to documented the purpose of all those tests

Reply:
For me the code is self explanatory and do not add something relevant. So it can be removed.